Missionframework/modules/11_permission/fnc/fn_permission_setupPermissionControls.sqf

```diff
@@ -0,0 +1,80 @@
+/*
+    KPLIB_fnc_permission_setupPermissionControls
+
+    File: fn_permission_setupPermissionControls.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2018-12-17
+    Last Update: 2018-12-29
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        Reads the player permissions and applies them to the dialog controls.
+
+    Parameter(s):
+        NONE
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
+// Dialog controls
+private _dialog = findDisplay 758011;
+private _ctrlPlayerList = _dialog displayCtrl 68740;
+
+// Colors
+private _red = [0.9,0,0,1];
+private _green = [0,0.8,0,1];
+private _white = [1,1,1,1];
```

Comment:
Static things like colors should be moved to defines. (`script_components.hpp`)

Not crucial right now but it might be worth remembering to move it out from here later.